docs/BreakingChanges/023 New (ambiguous) Dispatcher.Invoke overloads.md

```diff
@@ -0,0 +1,28 @@
+## 23: New (ambiguous) Dispatcher.Invoke overloads
```

Comment:
{
  "Id": "23",
  "Title": "New (ambiguous) Dispatcher.Invoke overloads",
  "Details": "The .NET Framework 4.5 adds new overloads to Dispatcher.Invoke that include a parameter of type System.Action. When existing code is recompiled, compilers may resolve calls to Dispatcher.Invoke methods that have a Delegate parameter as calls to Dispatcher.Invoke methods with an System.Action parameter. If a call to a Dispatcher.Invoke overload with a Delegate parameter is resolved as a call to a Dispatcher.Invoke overload with an System.Action parameter, the following differences in behavior may occur:\n\n- If an exception occurs, the Dispatcher.UnhandledExceptionFilter and Dispatcher.UnhandledException events are not raised. Instead, exceptions are handled by the UnobservedTaskException event.\n- Calls to some members, such as DispatcherOperation.Result, block until the operation has completed.",
  "Suggestion": "To avoid ambiguity (and potential differences in exception handling or blocking behaviors), code calling Dispatcher.Invoke can pass an empty object[] as a second parameter to the Invoke call to be sure of resolving to the .NET 4.0 method overload.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#wwf",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Windows.Threading.Dispatcher.Invoke(System.Delegate,System.Object[])",
    "M:System.Windows.Threading.Dispatcher.Invoke(System.Delegate,System.TimeSpan,System.Object[])",
    "M:System.Windows.Threading.Dispatcher.Invoke(System.Delegate,System.TimeSpan,System.Windows.Threading.DispatcherPriority,System.Object[])",
    "M:System.Windows.Threading.Dispatcher.Invoke(System.Delegate,System.Windows.Threading.DispatcherPriority,System.Object[])"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Windows.Threading.Dispatcher.Invoke(System.Delegate,System.Object[])
- [x] M:System.Windows.Threading.Dispatcher.Invoke(System.Delegate,System.TimeSpan,System.Object[])
- [x] M:System.Windows.Threading.Dispatcher.Invoke(System.Delegate,System.TimeSpan,System.Windows.Threading.DispatcherPriority,System.Object[])
- [x] M:System.Windows.Threading.Dispatcher.Invoke(System.Delegate,System.Windows.Threading.DispatcherPriority,System.Object[])